flow/project.py

```diff
@@ -3369,13 +3822,37 @@ def foo(job):
             A string to append to submissions can be any valid :meth:`FlowOperation.run` option.
         :type options:
             str
+        :param aggregator:
+            Aggregator function which aggregates jobs for this group.
+        :type aggregator:
+            callable
```

Comment:
We call it aggregator here but grouper elsewhere. We should be consistent in our names.

Reply:
`aggregator` is the right choice to go with. Will edit elsewhere.